src/lib/utils.js

```diff
@@ -0,0 +1,49 @@
+const { getProfiles } = require("aws-profile-utils");
```

Comment:
Don't use the `aws-profile-utils` package anymore, if you merge master to your branch you'll see the package has been removed and we have our own implementation which supports IAM roles (configured in `.aws/config` instead of `.aws/credential`).

Also, can we even assume someone is gonna be using the `switch-profile` command here? I guess that goes back to what do you show as the default when they're not. Showing `unknown` is just gonna be confusing I think. Maybe we'd just omitted it, or just be very specific - e.g. "the [default] profile" (but that might also be wrong, because the user can override that with the `AWS_PROFILE` env var...

Reply:
Makes sense. I was hoping to show a proper name in addition to the account ID, but your points are valid. Removing it and showing only the account ID